### DISCUSSION TITLE
How to use trpc with GO programming language?

### DISCUSSION THREAD

**Discussion by adamgajzlerowicz** [trpc/trpc#5922] [General]
We have an existing trpc server in our codebase that uses `https://github.com/trpc/trpc`.
I'd like to connect to the server from a client written in GO.

I tried generating code with `https://github.com/trpc-group/trpc-cmdline` which was very promising but it seems that the libraries are not compatible with each other.

Is there a recommended way I can generate the go code to access the data?

Thanks

**Comment by mmkal** [comment]
TIL about trpc.group - that's an unrelated tool that inexplicably chose an existing name for a pretty well established thing, with fairly significant overlap!

**Reply by adamgajzlerowicz** [reply]
Thank you for your input.
